src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -376,18 +373,22 @@ object EntityClient {
 
   sealed trait EntityType {
     val entityPath: String
+    val entityName: String
```

Comment:
Maybe  `entityTypeAcronym`, `entityTypeAsAcronym`, `entityTypeAcronymised`, `entityTypeAbbreviated`, `entityTypeAbr`, `entityTypeShort`, `entityTypeShortend`?

Reply:
`entityTypeShort` is Ok. The rest are a bit long to scan and  I don't know how I feel about abbreviating the word abbreviated :smile: